chia/full_node/mempool.py

```diff
@@ -6,27 +6,24 @@
 
 from sortedcontainers import SortedDict
 
-from chia.full_node.bitcoin_fee_estimator import create_bitcoin_fee_estimator
-from chia.full_node.fee_estimation import FeeMempoolInfo
+from chia.full_node.fee_estimation import FeeMempoolInfo, MempoolInfo
 from chia.full_node.fee_estimator_interface import FeeEstimatorInterface
 from chia.types.blockchain_format.coin import Coin
 from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.types.clvm_cost import CLVMCost
-from chia.types.fee_rate import FeeRate
 from chia.types.mempool_item import MempoolItem
 from chia.util.ints import uint64
 
 
 class Mempool:
-    def __init__(self, max_size_in_cost: int, minimum_fee_per_cost_to_replace: uint64, max_block_cost_clvm: uint64):
+    def __init__(self, mempool_info: MempoolInfo, fee_estimator: FeeEstimatorInterface):
```

Comment:
I like using a `FeeEstimatorInterface` instead of the mempool creating the fee estimator

Reply:
Thanks!